src/components/DailyTasks.spec.js

```diff
@@ -0,0 +1,15 @@
+import {render, screen} from '@testing-library/react';
+import DailyTasks from './DailyTasks';
+describe('DailyTasks', () => {
+  it('renders heading, description and an unsorted list', () => {
+    render(<DailyTasks />);
+
+    const heading = screen.getByRole('heading', {name: 'täglich'});
+    const description = screen.getByText(/Aufgaben/i);
+    const list = screen.getByRole('list');
+
+    expect(heading).toBeInTheDocument();
+    expect(description).toBeInTheDocument();
+    expect(list).toBeInTheDocument();
+  });
```

Comment:
Looks good to me. You might add another test suite (i.e. another file for DailyTaskList.spec.js) in order to check whether the content of the list is the same you expect. It would be good if you pass the content of each list item as a prop (see my comment above), then you can test this dump component no matter how it might change in the future.